packages/serverpod_serialization/lib/src/exceptions.dart

```diff
@@ -0,0 +1,14 @@
+/// This is `ServerpodException` that can be used to pass Domain exceptions from the Server to the Client
+///
+/// You can `throw ServerpodException()`
+///
+/// Based on issue [#486](https://github.com/serverpod/serverpod/issues/486)
+class ServerpodException implements Exception {
```

Comment:
My bad, but I think a better name here would be `SerializableException`. It just would be a bit more clear.